pages/windows/vswhere.md

```diff
@@ -0,0 +1,20 @@
+# vswhere
+
+> Locate Visual Studio 2017 and newer installations.
+> More information: <https://github.com/microsoft/vswhere>.
+
+- Find the path of vcvarsall.bat to set environment variables:
```

Comment:
I'm not sure what this `to set the environment variables` part does here. Could you please explain this a bit?

Reply:
`vsvarsall` is used to set environment variables. See #7439